src/all/exhentai/src/eu/kanade/tachiyomi/extension/all/exhentai/MetadataCopier.kt

```diff
@@ -0,0 +1,86 @@
+package eu.kanade.tachiyomi.extension.all.exhentai
+
+import eu.kanade.tachiyomi.source.model.SManga
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+private const val EH_ARTIST_NAMESPACE = "artist"
+private const val EH_AUTHOR_NAMESPACE = "author"
+
+private val ONGOING_SUFFIX = arrayOf(
+    "[ongoing]",
+    "(ongoing)",
+    "{ongoing}",
```

Comment:
Is this list exhaustive? If it's arbitrary text you could look into using the various bracket pairs defined in #5172.